en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -44,47 +44,84 @@ class MyDataObject extends DataObject
 > [!TIP]
 > It is typically considered a good practice to wrap your modifications in a call to [`beforeUpdateCMSFields()`](api:SilverStripe\ORM\DataObject::beforeUpdateCMSFields()) - the `updateCMSFields()` extension hook is already triggered by `parent::getCMSFields()`, so this is how you ensure any new fields are added before extensions update your fieldlist.
 
-To fully customise your form fields, start with an empty FieldList.
+To define the form fields yourself without using scaffolding, use the `tabsOnly` option in [`DataObject.scaffold_cmsfields_settings`](api:SilverStripe\ORM\DataObject->scaffold_cmsfields_settings). See [scaffolding options](#scaffolding-options) for details.
```

Comment:
This is a new best practice - it's always best to call the parent method when you can, and now you can.